.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_TINK_LINK_PAYMENT_CLIENT_ID=
+TINK_LINK_PAYMENT_CLIENT_SECRET=
```

Comment:
Are these all the variables this project uses? Lets list them all so we have a better idea what needs to be setup prior to running it.

Reply:
After review, as this is a one-time payment, the only thing the client needs is the Client Id & Secret to create a session with Tink Link. 

We could potentially parameterise something like the Product information, as it's hard coded a the moment. Which would mean the demo could be quickly repurposed via the config file, instead of changing server values.

```
  destinations: [
    {
      accountNumber: env.process.BANK_ACCOUNT_NUMBER,
      type: env.process.PAYMENT_TYPE
      reference: env.process.REFERENCE
    },
  ],
  recipientName: env.process.RECIPIENT_NAME
  sourceMessage: env.process.SOURCE_MESSAGE
  remittanceInformation: {
    type: env.process.RM_INFO_TYPE
    value: env.process.RM_INFO_VALUE
  },
```

Let me know your thoughts

Reply:
I was thinking about moving all existing env variables in to this new `.env` variables, not creating a new ones. If `REACT_APP_TINK_LINK_PAYMENT_CLIENT_ID` and `TINK_LINK_PAYMENT_CLIENT_SECRET` is the only two, lets merge this.

Reply:
@rtelicak thanks! I don't have merge perms still but happy to see this move forward!